system/src/control/config.cpp

```diff
@@ -171,41 +171,25 @@ int getDeviceMode(ctrl_request* req) {
 }
 
 int setDeviceSetupDone(ctrl_request* req) {
-#if HAL_PLATFORM_DCT_SETUP_DONE
+    // This functionality is currently deprecated.
+    // Do not perform any DCT accesses. Instead return an appropriate error code
```

Comment:
I'd probably leave `HAL_PLATFORM_DCT_SETUP_DONE` conditional, or get rid of it completely everywhere.